src/alert/types.ts

```diff
@@ -14,4 +14,11 @@ export interface AlertProps extends React.HTMLAttributes<HTMLDivElement> {
     showIcon?: boolean | undefined;
     actionLinkIcon?: JSX.Element | undefined;
     sizeType?: AlertSizeType | undefined;
+    /** Custom alert icon to be rendered, replacing the default alert icons */
+    customIcon?: JSX.Element | undefined;
+    /** 
+     * Specifies the maximum height of the element to be
+     * displayed outside of the collapsible zone, in px 
+    */
+    collapsedHeight?: number | undefined;
```

Comment:
naming can be consistent with the notification banner if possible

```suggestion
    maxCollapsedHeight?: number | undefined;
```